projects/plugins/jetpack/extensions/blocks/mailchimp/view.scss

```diff
@@ -8,10 +8,21 @@
 		}
 	}
 
-	.wp-block-jetpack-button {
+	p {
 		margin-bottom: 1em;
 	}
 
+	input {
```

Comment:
This looks good to me visually, but I can see that it's overriding the theme provided styles when using the Barnsbury theme in wpcom. Without the change applied, the theme's CSS targets the input field and sets it to 16px padding:

![image](https://user-images.githubusercontent.com/14988353/115504086-b63d0080-a2ba-11eb-8e59-0da4c42c55bf.png)

With the change applied, this CSS overrides it and sets the padding to `14px`:

![image](https://user-images.githubusercontent.com/14988353/115504176-dbca0a00-a2ba-11eb-8571-f713a6fb1c68.png)

With this particular theme the difference isn't all that noticeable, but I imagine other themes might diverge a bit more in their padding? That said, I can't quite think how we'd reduce the specificity here to allow the theme styles to override it 🤔

Reply:
I agree, I'd prefer to not override the themes here. Maybe this particular case is a reasonable one to defer -- remove the styling here and instead ask the block themes to provide a better base style?

Reply:
That's a good point, I notice that the Search block in core also doesn't apply padding by default in TT1-blocks, so maybe it's better to leave it (aside from the `width: 100%` that you've added).

Search block:

![image](https://user-images.githubusercontent.com/14988353/115674235-cb845e80-a390-11eb-9525-e4ea7a29ac28.png)

Unfortunately I still haven't had much luck investigating the placeholder issue in my environment. I'll try again tomorrow, as I'd like to get to the bottom of what's causing it. Apologies for holding up the review!

Reply:
No worries, thank you!

Reply:
It turns out the weird placeholder issue I encountered occurs only when you view the front end of the site while logged in, with a site that doesn't have a paid plan, and you also have a block on the page that displays an upgrade nudge. I've put together a fix in: https://github.com/Automattic/jetpack/pull/19639